console/src/assets/i18n/en.json

```diff
@@ -154,54 +153,54 @@
                 "VALID": "validated",
                 "RESEND": "Resend verification message"
             },
-            "RESENDCODE": "Code senden",
+            "RESENDCODE": "Resend Code",
             "ENTERCODE":"Verify",
-            "ENTERCODE_DESC":"Verify number"
+            "ENTERCODE_DESC":"Verify Code"
         },
-        "ADDRESS": {
-            "TITLE": "Address",
-            "STREET": "Street/No",
-            "LOCALITY": "Place",
-            "POSTAL_CODE": "Postal Code",
-            "REGION": "Region",
-            "COUNTRY": "Country"
+        "GRANTS": {
+            "TITLE": "User Grants",
+            "DESCRIPTION": "Give this user control of certain projects",
+            "CREATE": {
+                "TITLE": "Create User Grant",
+                "DESCRIPTION": "Search for the organisation, the project and the corresponding project roles!"
+            }
         },
         "VALIDATION": {
-            "INVALIDPATTERN": "The password must consist of at least 8 characters and contain a capital letter, a special character and a number. The maximum length is 72.",
+            "INVALIDPATTERN": "## TODO - Replace with Password Policy text - TODO ##",
             "REQUIRED": "The input field is empty",
             "MINLENGTH":"The password has to be at least {{requiredLength}} characters long!",
-            "NOEMAIL":"Username can not be an email",
-            "UPPERCASEMISSING":"An uppercase character is needed!",
-            "LOWERCASEMISSING":"A lowercase character is needed!",
+            "NOEMAIL":"Username can not be an email address!",
+            "UPPERCASEMISSING":"The password must include a uppercase character!",
+            "LOWERCASEMISSING":"The password must include a lowercase character!",
             "SYMBOLERROR":"The password must include a symbol!",
             "NUMBERERROR":"The password must include a number!"
         },
         "STATE": {
-            "0":"unbekannt",
-            "1":"aktiv",
-            "2":"abgelaufen"
+            "0":"Unknown",
+            "1":"Active",
+            "2":"Expired"
         },
         "SEARCH":{
-            "FOUND":"found"
+            "FOUND":"Found"
         },
         "TARGET": {
-            "SELF":"You will get users of your organization. If you want to add an external user",
-            "EXTERNAL":"To get users of your organization",
+            "SELF":"You will get users of your organisation. If you want to add an external user", #DAFUQ
+            "EXTERNAL":"To get users of your organisation", #DAFUQ
```

Comment:
@fgerschwiler @peintnermax i have no clue what this does 😁

Reply:
if youd like to grant an user of an other organisation you need to click the link, because its another search

Reply:
```suggestion
            "SELF":"If you want to grant a user of an other organisation",
            "EXTERNAL":"To grant a user of your organisation",
```